app/controllers/spree/api/v2/display_products_controller.rb

```diff
@@ -23,16 +22,6 @@ def create
           end
         end
 
-        def update
```

Comment:
@mausamp why are you deleting this?

Reply:
There was no any route defined for the update action, so it was deleted.